tests/e2e/contracts/user_storage.py

```
@@ -0,0 +1,24 @@
+    # constructor
+    def __init__(self):
+        pass
+
+    # read methods must be annotated
+    @gl.public.view
+    def get_complete_storage(self) -> dict[str, str]:
+        return {k.as_hex: v for k, v in self.storage.items()}
+
+    @gl.public.view
+    def get_account_storage(self, account_address: str) -> str:
+        return self.storage[Address(account_address)]
```

Ruff (0.11.9)

`Address` may be undefined, or defined from star imports

(F405)